tests/RobotFramework/README.md

```diff
@@ -158,16 +158,15 @@ Checkout the [dev container instructions](https://thin-edge.github.io/thin-edge.
 
 6. Build the container image used by the default `docker` test adapter
 
-    The following command will use the debian packages built using the latest packages from the main branch (provided by Cloudsmith.io)
-
     ```sh
-    invoke clean build
+    just release
+    invoke build
     ```
 
-    Or you can use the locally built debian packages (assuming you have already built them)
+    Or you can use the latest packages from the main branch (provided by Cloudsmith.io)
```

Comment:
Packages from the `main` channel or the `release` channel? The latter, right?

Reply:
The `main` channel is the default (when not using local builds). Original idea was to make it easy to test against the latest build in the main branch before doing an official release, however in practice this feature is rarely used.